modules/core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/api/validation/TestDocumentValidationService.java

```diff
@@ -621,8 +629,16 @@ public void testDocumentValidationReportMessage() {
         assertTrue(report.hasError());
         assertEquals(MESSAGE_FOR_USERS_FIRSTNAME, report.toString());
 
+        violation = report.asList().get(0);
+        if (violation instanceof ConstraintViolation) {
+            cv = (ConstraintViolation) violation;
+        }
+
         e = new DocumentValidationException(report);
-        expectedMessage = String.format(DocumentValidationException.MESSAGE_SINGLE, MESSAGE_FOR_USERS_FIRSTNAME);
+
+        //changed in 11.5 since the message includes the property name (NXP-29983)
+        expectedMessage = String.format(DocumentValidationException.MESSAGE_SINGLE, cv.getPath().get(0),
+                MESSAGE_FOR_USERS_FIRSTNAME);
```

Comment:
In expected message it's better to have the explicit value as a string constant instead of `cv.getPath().get(0)` that hides what we really are expecting.

Reply:
So here really we want a full string constant for the entire expected message, no reuse of `DocumentValidationException.MESSAGE_SINGLE` (which means the previous code was bad).